### PR TITLE
chore: Send default environmentId in header of all api call

### LINKED ISSUE #23766: [Bug]: Send default environmentId in header of all api call
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Send default environmentId in header of all api call to `unused_env` in CE

### Steps To Reproduce

Check header of all api calls

### Public Sample App

_No response_

### Environment

Production

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by ChandanBalajiBP**
#### PR fixes following issue(s)
Fixes #23766
>
#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)
>
>
>
#### How Has This Been Tested?
This changes is w.r.t #1521 PR in EE. Which will be tested in regression. 
- [x] Regression
>
>
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flagx

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/sagas/ErrorSagas.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ import {
 import store from "store";
 
 import * as Sentry from "@sentry/react";
-import { axiosConnectionAbortedCode } from "api/ApiUtils";
+import { axiosConnectionAbortedCode } from "ce/api/ApiUtils";
```
Comment (ayushpahwa) [inline app/client/src/sagas/ErrorSagas.tsx:32]: Can we change this to @appsmith/apiUtils here?

**Review by ayushpahwa — CHANGES_REQUESTED** [review]

**Review by ayushpahwa — APPROVED** [review]

**Comment by ChandanBalajiBP** [comment]
This PR will be test in regression, hence adding test plan approved label myself.

**Pull request merged**